data_processing/grid_demand_objective_function.ipynb

```diff
@@ -0,0 +1,817 @@
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>2018-07-01 00:00:00</th>\n",
+       "      <td>74</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2018-07-01 01:00:00</th>\n",
+       "      <td>73</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2018-07-01 02:00:00</th>\n",
+       "      <td>72</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2018-07-01 03:00:00</th>\n",
+       "      <td>72</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2018-07-01 04:00:00</th>\n",
+       "      <td>72</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>...</th>\n",
+       "      <td>...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2019-06-30 19:00:00</th>\n",
+       "      <td>68</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2019-06-30 20:00:00</th>\n",
+       "      <td>68</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2019-06-30 21:00:00</th>\n",
+       "      <td>69</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2019-06-30 22:00:00</th>\n",
+       "      <td>68</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2019-06-30 23:00:00</th>\n",
+       "      <td>67</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>8760 rows × 1 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                    temperature\n",
+       "Date and Time                  \n",
+       "2018-07-01 00:00:00          74\n",
+       "2018-07-01 01:00:00          73\n",
+       "2018-07-01 02:00:00          72\n",
+       "2018-07-01 03:00:00          72\n",
+       "2018-07-01 04:00:00          72\n",
+       "...                         ...\n",
+       "2019-06-30 19:00:00          68\n",
+       "2019-06-30 20:00:00          68\n",
+       "2019-06-30 21:00:00          69\n",
+       "2019-06-30 22:00:00          68\n",
+       "2019-06-30 23:00:00          67\n",
+       "\n",
+       "[8760 rows x 1 columns]"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "df = pd.read_csv(\"2019_uiuc_electricity_demand.csv\",\n",
+    "                 skiprows=1,\n",
+    "                 parse_dates=True,\n",
+    "                 usecols=['Date and Time', 'Value (KW)'],\n",
+    "                 index_col = ['Date and Time'])\n",
+    "demd2019 = df.rename(columns={\"Value (KW)\":\"kW\"})\n",
+    "\n",
+    "df = pd.read_csv(\"willard_weather_data.csv\",\n",
+    "                 parse_dates=True, usecols=['DATE', 'HourlyWetBulbTemperature'],\n",
+    "                 index_col = ['DATE'])\n",
+    "df = df.rename(columns={\"HourlyWetBulbTemperature\":\"temperature\"})  # temperature in F\n",
```

Comment:
```suggestion
    "df = df.rename(columns={\"HourlyWetBulbTemperature\":\"wb_temperature\"})  # wet bulb temperature in F\n",
```

Reply:
I want to leave this generic in case future modifications include another type of temperature.
However, I will accept the changes on the plots' labels/titles.